scripts/api_review/generate_review_files.py

```diff
@@ -0,0 +1,82 @@
+import re
```

Comment:
wanted to still further improve this - will probably move the method parsing logic out and generate a .json file with it that is then used by this script. This makes it also possible to supply additional info such as markdown-links for docs / definition

Reply:
I wouldn't do such things, better keep it as simple as it is.